src/settings/settings.js

```diff
@@ -3,9 +3,11 @@ const themeLegend = document.getElementById('themeTitle');
 const defaultThemeLabel = document.getElementById('default_label');
 const darkThemeLabel = document.getElementById('dark_label');
 
+/* eslint-disable no-unsanitized/property */
```

Comment:
I just disabled the rule for these 3 lines. Not sure if there is some "sanitization" module we wanted to use or something to make sure there aren't any potential XSS leaks from the l10n files.